SocialFilter/sections/popups.txt

```diff
@@ -8,6 +8,10 @@
 ||mailget.net^$third-party
 ||my.hellobar.com^$third-party
 !
+!+ PLATFORM(android, ios, ext_android_cb)
+unn.com.ua##.fancybox-overlay
+!+ PLATFORM(android, ios, ext_android_cb)
+unn.com.ua##.fancybox-wrap
```

Comment:
Same issue as reported for those devices then.

Please try these rules:

##body > .fancybox-opened
##body > .fancybox-opened + .fancybox-overlay

Reply:
`##body > .fancybox-opened + .fancybox-overlay` hide overlay, but not a popup.

`unn.com.ua##body > .fancybox-opened`
`unn.com.ua##body > .fancybox-overlay` these are ok, but i get the same problem (now in mobile version) - can't open photo from the article in issue.

Reply:
The first is hiding the popup for me, 2nd one overlay. But not the photo on mobile for me.

Else please set the cookie which gets created when closing the initial popup on mobile.

Reply:
Please check this rules on your device. It seems ok. Without `#%#` rule I noticed a popup for 1 second.

`unn.com.ua##body > .fancybox-opened`
`unn.com.ua##body > .fancybox-opened + .fancybox-overlay`
`unn.com.ua#%#document.cookie = "scps=1";`

Reply:
As mentioned first and 2nd rule are working fine for me without the need of cookie rule.
If they are malfunctioning for you, cookie is enough for android, while ios and cb do not support cookie rule.